lib/hci-socket/gatt.js

```diff
@@ -153,7 +153,9 @@ Gatt.prototype.onAclStreamEncrypt = function(encrypt) {
   if (encrypt) {
     this._security = 'medium';
 
-    this.writeAtt(this._currentCommand.buffer);
+    if(this._currentCommand) {
```

Comment:
I assume during testing that you ran into cases where where onAclStreamEncrypt was called, but this._currentCommand was null, which through an error when trying to write its buffer?

Reply:
That is correct.